src/main/scala/scalismo/statisticalmodel/asm/ActiveShapeModel.scala

```diff
@@ -91,6 +98,39 @@ case class ActiveShapeModel(statisticalModel: StatisticalMeshModel, profiles: Pr
     }
   }
 
+  /**
+   * Returns the mean mesh of the shape model, along with the mean feature profiles at the profile points
+   */
+  def mean(): ASMSample = {
+    val meanProfilePoints = pointIds.map(id => statisticalModel.mean(id))
+    val meanFeatures = profiles.values.map(_.mean).toIndexedSeq
+    val featureField = DiscreteFeatureField(meanProfilePoints zip meanFeatures)
+    ASMSample(statisticalModel.mean, featureField)
+  }
+
+  /**
+   * Returns a random sample mesh from the shape model, along with randomly sampled feature profiles at the profile points
+   */
+  def sample(): ASMSample = {
+    val sampleMesh = statisticalModel.sample
+    val randomProfilePoints = pointIds.map(id => sampleMesh(id))
+    val randomFeatures = profiles.values.map(_.sample).toIndexedSeq
+    val featureField = DiscreteFeatureField(randomProfilePoints zip randomFeatures)
+    ASMSample(sampleMesh, featureField)
+  }
+
+  /**
+   * Utility function that allows to randomly sample different feature profiles, while keeping the profile points
+   * Meant to allow to easily inspect/debug the feature distribution
+   */
+
+  def sampleFeaturesOnly: ASMSample = {
```

Comment:
This should explicitly be defined as sampleFeaturesOnly() -- i.e., with parentheses. In contrast, the mean() method *could* (I'm not saying "should", I don't know what's better) be defined in this accessor-like style without parentheses.